submit.yml.erb

```diff
@@ -46,3 +47,92 @@ script:
     <%- slurm_args.each do |arg| %>
     - "<%= arg %>"
     <%- end %>
+<% elsif cluster =~ /kubernetes/
+   if node_type =~ /owens/
+     compute_cluster = "owens"
+     apps_path = "/usr/local"
+   elsif node_type =~ /pitzer/
+     compute_cluster = "pitzer"
+     apps_path = "/apps"
+   end
+   mounts = {
+     'home'    => OodSupport::User.new.home,
+     'project' => '/fs/project',
+     'scratch' => '/fs/scratch',
+     'ess'     => '/fs/ess',
+   }
+-%>
+---
+script:
+  accounting_id: "<%= account %>"
+  wall_time: "<%= bc_num_hours.to_i * 3600 %>"
+  <%- if node_type =~ /gpu/ -%>
+  gpus_per_node: 1
+  <%- end -%>
+  native:
+    container:
+      name: "jupyter"
+      image: "docker-registry.osc.edu/ondemand/ondemand-base-rhel7:0.1.0"
+      image_pull_policy: "IfNotPresent"
+      command: ["/bin/bash","-l","<%= staged_root %>/job_script_content.sh"]
+      restart_policy: 'OnFailure'
+      env:
+        NB_UID: "<%= Etc.getpwnam(ENV['USER']).uid %>"
+        NB_USER: "<%= ENV['USER'] %>"
+        NB_GID: "<%= Etc.getpwnam(ENV['USER']).gid %>"
+        CLUSTER: "<%= compute_cluster %>"
+        KUBECONFIG: "/dev/null"
+      port: "8080"
+      cpu: "<%= num_cores %>"
+      memory: "<%= num_cores.to_i * 4 %>Gi"
+    mounts:
+    <%- mounts.each_pair do |name, mount| -%>
+      - type: host
+        name: <%= name %>
+        host_type: Directory
+        path: <%= mount %>
+        destination_path: <%= mount %>
+    <%- end -%>
+      - type: host
+        name: munge-socket
+        host_type: Socket
+        path: /var/run/munge/munge.socket.2
+        destination_path: /var/run/munge/munge.socket.2
+      - type: host
+        name: slurm-conf
+        host_type: Directory
+        path: /etc/slurm
+        destination_path: /etc/slurm
+      - type: host
+        name: sssd-pipes
+        host_type: Directory
+        path: /var/lib/sss/pipes
+        destination_path: /var/lib/sss/pipes
+      - type: host
+        name: sssd-conf
+        host_type: Directory
+        path: /etc/sssd
+        destination_path: /etc/sssd
+      - type: host
+        name: nsswitch
+        host_type: File
+        path: /etc/nsswitch.conf
+        destination_path: /etc/nsswitch.conf
+      - type: host
+        name: lmod-init
+        host_type: File
+        path: /apps/<%= compute_cluster %>/lmod/lmod.sh
+        destination_path: /etc/profile.d/lmod.sh
+      - type: host
+        name: intel
+        host_type: Directory
+        path: /nfsroot/<%= compute_cluster %>/opt/intel
+        destination_path: /opt/intel
+      - type: host
+        name: apps
+        host_type: Directory
+        path: /apps/<%= compute_cluster %>
+        destination_path: <%= apps_path %>
```

Comment:
Looks like we'll have to mount the `support` user's home directory too `/users/PZS0645/support`.  There are all sorts of class module config files in there.

Reply:
There is a hash of static mounts where the path and destination are identical , so I just did this:

```
'support' => OodSupport::User.new('support').home,
```

I also made that change on rstudio app because I noticed that the path is referenced in a file.

Reply:
:facepalm: not sure why I mentioned it here, I was testing Rstudio and meant to mention it there.